docker/entrypoints/dev.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -euo pipefail
+
+./manage.py migrate
+./manage.py ensure_groups
+./manage.py collectstatic --no-input --clear | grep -v '^Deleting '
```

Comment:
Can we do `just collectstatic` to avoid this if it's not needed?

Reply:
I gave this a solid shot and couldn't find a way to pair the just and docker stacks yet so I've left it as it is a the pragmatic way forward, and duplicated the pytest & coverage commands into the docker-compose.yaml.

Reply:
I've fixed this, as part of solving the staticfiles ownership conflict